include/dlaf/matrix/tile.h

```diff
@@ -362,12 +364,16 @@ namespace internal {
 template <class T, Device D>
 pika::future<Tile<T, D>> createSubTile(const pika::shared_future<Tile<T, D>>& tile,
                                        const SubTileSpec& spec) {
-  return pika::dataflow(
-      pika::launch::sync, [](auto tile, auto spec) { return Tile<T, D>(tile, spec); }, tile, spec);
+  namespace ex = pika::execution::experimental;
+  auto f = [](auto tile, auto spec) { return Tile<T, D>(tile, spec); };
```

Comment:
What about something like this?

```suggestion
  auto f = [](const Tile<T, D>& tile, const SubTileSpec spec) { return Tile<T, D>(tile, spec); };
```

Reply:
Note, it's `[](pika::shared_future<Tile<T, D>>&& tile, ...)` not just `Tile<T, D>`, so slightly more verbose than what you suggested. I still like being explicit, but let me know if you'd prefer the `auto` after all.

Reply:
I like being explicit too.

My tendency in `auto` usage is to use it when the type is already clear in the line.